apps/system/js/app_text_selection_dialog.js

```diff
@@ -9,6 +9,11 @@
   // states which are shared by apps should be put in this class.
   var AppTextSelectionDialogGlobalStates = function() {
     this._hasCutOrCopiedTimeoutId = null;
+    this._isPrefOn = true;
+    this._appTSDs = new Set();
```

Comment:
One little small change (sorry): `s/_appTSDs/appTSDs/g` since we intend to allow external access of the set.